### PR TITLE
Added Minetest package

### PR DESCRIPTION
**Pull request by kunisensei92**
Simply added Minetest game package for testing.

### CODE REVIEW

File: packages/minetest.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+class Minetest < Formula
```
Comment (uberhacker) [inline packages/minetest.rb]:
```
+ require 'package'
+
- class Minetest < Formula
+ class Minetest < Package
```

File: packages/minetest.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  desc "Free, open source voxel game engine and game"
```
Comment (uberhacker) [inline packages/minetest.rb]:
```
- desc "Free, open source voxel game engine and game"
+ description "Free, open source voxel game engine and game"
```

File: packages/minetest.rb [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  homepage "https://www.minetest.net/"
+  revision 1
```
Comment (uberhacker) [inline packages/minetest.rb]:
```
- revision 1
+ version "1"
```

File: packages/minetest.rb [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      sha256 "ea914555949b4faccda5e13143cb021d2f9a5fa19abd1f5e4b7b65004cbd2b5a"
+    end
+  end
+
+  bottle do
+    sha256 "557c9618fd6f9c16c13cf74ef76e3923e71334ba68c3100a60539d331edb802d" => :high_sierra
+    sha256 "bd1bb5708000861f5406b4e95355f4474b372136e96db374076c29dc9464fa74" => :sierra
+    sha256 "8ba27901ec54f9f141c3afb94406d22ec49f907f4e1c873375b5338703e1d3e2" => :el_capitan
+    sha256 "a56489395aa0d13f322ced52167d8e46974bfd6e8302a0e6b0335df3082eed4e" => :yosemite
+  end
```
Comment (uberhacker) [inline packages/minetest.rb]: This looks like a direct copy from Homebrew.  This entire section will need to be changed.  We do not use bottles.  We follow a different structure.  Please check out some of the existing packages in https://github.com/skycocker/chromebrew/tree/master/packages for examples on the proper syntax.

**Comment by uberhacker** [comment]
@kunisensei92: Thank you for the new package, however, we follow a different practice than Homebrew.  Please see the inline comments in the `Files changed` section.

**Comment by flyingP0tat0** [comment]
Does this package actually work on ChromeOS?

**Comment by uberhacker** [comment]
Closing since this was obviously just copy/pasted from Homebrew.

**Pull request closed without merging**